ietf/meeting/models.py

```diff
@@ -419,6 +419,15 @@ def group_at_the_time(self, group):
         self.cached_groups_at_the_time[group.pk] = new_item
         return new_item
 
+    def use_attended(self):
+        if not hasattr(self,'cached_use_attended'):
+            try:
+                meeting_118 = Meeting.objects.get(number="118")
```

Comment:
The date isn't going to move - hardcoding it would be better than hitting the database